### PR TITLE
bug/EUI-7687 package json file changes

### PR DESCRIPTION
**Pull request by Kartheek-S**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-7874

### Change description ###
Collections not displaying correctly

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by GaryScottHMCTS — DISMISSED** [review]

**Review by bradley-hmcts — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Comment by deanbullock-hmcts** [comment]
Trying to view this change in the preview environment but the build has failed. Is this a non-issue and can be ignored? @Mo-Lala84

File: package.json [hunk 1]
```diff
@@ -74,7 +74,7 @@
     "@angular/platform-browser-dynamic": "^11.2.14",
     "@angular/router": "^11.2.14",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "6.13.9-rc1",
```
Comment (deanbullock-hmcts) [inline package.json:77]: This need changing to reflect latest toolkit changes?

Comment (deanbullock-hmcts) [reply package.json:77]: 6.13.10-show-condition-code-fix-2

**Review by deanbullock-hmcts — DISMISSED** [review]

**Pull request closed without merging**